en/frames_plane/turbo_timber_evolution.md

```diff
@@ -0,0 +1,229 @@
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/pre_mount_install.jpg)
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/pixhawk_mount.jpg)
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/mount_fit_test.jpg)
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/pixhawk_wired.jpg)
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/top_down.jpg)
+
+First, the Pixhawk 4 Mini was placed in the lower mount.
+Hot glue was added to rigidly connect the fcu to the mount with two zipties providing additional security.
+The standoff mounts for the upper mount were installed, and the bolts securely tightened.
+Once the lower mount is installed, these screws are inaccessible, so attention was paid to making sure they were tightt enough so they cannot back out. 
+
+## Electrical
+
+### Power
+
+The Holybro power module was wired inline with the ESC.
+A spare 16awg power lead was also broken out, terminated to an XT30.
+This spare lead will be used to provide power to the Caddx Vista, but could also be connected to a splitter to power more peripherals.
+Power for the servo and lighting will be provided by the "BEC" power supply in the ESC. 
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/power_module.jpg)
+
+The TTE is very flexible when it comes to battery options.
+I use both a 3.6Ah 4S Turnigy pack as well as a Upgrade Energy 4s2p liion pack.
+While the 3.6Ah LiPo is inexpensive, nearly twice the flight time (24 minutes vs 12 minutes) can be acheived with the Upgrade Energy Liion pack.
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/batteries.jpg)
+
+### Servos
+
+Servos were wired to the flight computer in order of aileron, elevator, rudder, throttle, flaps, and FPV pan.
+An additional power plug for the lighting system needs to also be installed, but it does not carry a servo signal so it can be put on any spare channel.
+
+The [Acutator Configuration](../config/actuators.md) screen is shown below.
+
+![Turbo Timber Evolution Build](../../assets/airframes/fw/turbo_timber_evolution/qgcactuators.png)
+
+Servo endpoints were obtained by using a servo tester to determine the servo PWM pulse width to reach the max travel of each surface in each direction.
+
+
+### Config & Debug
+
+Access to the Pixhawk 4 Mini requires removal of the upper mount.
+While this isn't too difficult, it was a consideration for wanting to streamline debugging in the field.
+A short right angle USB micro extension was used to allow easy access to the Pixhawk 4 Mini's USB interface.
+The USB-A end of this cable was left dangling in the battery bay.
+Similarly, a JST PH to std spaced headers adapter was made, and it was also left easily accessible in the battery bay. 

```

Comment:
Would be good to have a picture of the adapters. Wouldn't block this going in though.

Reply:
Yeah I wished I had taken one one. Unfortunately, they're buried in the airplane now, so that was the only reason I didn't include a shot.

Reply:
Bugger!